code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -356,6 +356,7 @@
 
 	to_chat(H, "<span class='biggerdanger'>You have been Enthralled by [user]. Follow [user.p_their()] every command.</span>")
 	to_chat(user, "<span class='warning'>You have successfully Enthralled [H]. <i>If [H.p_they()] refuse[H.p_s()] to do as you say just adminhelp.</i></span>")
+	H.Weaken(2)
```

Comment:
nitpick but I mean the Stun() command not the weaken, we don't want to drop them we just want to lock their controls so they can't immediately attack back